libs/api/src/main/java/de/cyface/api/Authorizer.java

```diff
@@ -215,6 +211,7 @@ public Future<Set<User>> loadAccessibleUsers(final JsonObject principal) {
         return promise.future();
     }
 
+    // TODO: @Armin: Why is this here and not inside the UserRetriever (since it retrieves users).
```

Comment:
Please see here.

Reply:
Because this method is only used by the `Authorizer` which loads the accessible users so that all API Endpoint handlers can return data accessible by the requester. 

I moved the `libs/api` classes to a new repository: https://github.com/cyface-de/api/pull/1/commits/944308cb9baaa8c5dc1998d7aa1b15266a90a5d4

I changed the method to be package-private there, as the test which was the reason why this had to be public now is also part of `libs/api`.

Reply:
**Attention:**

I had to delete the artifacts `de.cyface.api` from the `data-collector` repository or else I cannot publish these artifacts from the new repository `api`.